app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,40 @@
+class MerchantsController < ApplicationController
+  def index
+    @merchants = Merchant.all
+  end
+
+  def new
+    @merchant = Merchant.new
```

Comment:
since your view that's rendered with the new action should be a form that creates a merchant, you actually don't need anything in this method. The idea is that the route "/merchants/new" will route to "merchants#new" (merchants controller and then to the new method), and then this new method will directly render a view. We don't need anything other than a form to create the view, so we don't need anything in this method.